data/ga/ACCESSCONTROL-ROLES/rolesdesignations.json

```diff
@@ -67,6 +67,30 @@
 			"designationCode": "MD",
 			"designationName": "Managing Director",
 			"active": true
+		},
+		{
+			"rolecode": "ESTD_VERIFIER_AM",
+			"designationCode": "AM",
+			"designationName": "Assistant Manager",
+			"active": true
+		},
+		{
+			"rolecode": "ESTD_VERIFIER_DGM",
+			"designationCode": "DGM",
+			"designationName": "Deputy General Manager",
+			"active": true
+		},
+		{
+			"rolecode": "ESTD_VERIFIER_GM",
+			"designationCode": "GM",
+			"designationName": "General Manager",
+			"active": true
+		},
+		{
+			"rolecode": "ESTD_APPROVER",
+			"designationCode": "MD",
+			"designationName": "Managing Director",
+			"active": true
```

Comment:
Why you need separate Role code for Estate MD?

I think MD is not associated with any Department!!!

If you want to use anywhere role for MD, please use the exsisting role for MD, which you can find above in this fine Only.
`{
			"code": "GIDC_MD",
			"name": "Goa IDC Managing Director",
			"description": "Goa IDC Managing Director (MD)"
}`

Reply:
Okay, we will remove the 'estd_approver'  and use 'gidc_md'  for the same.